ci/vars-zuul-common.yml

```diff
@@ -0,0 +1,6 @@
+---
+namespace: "service-telemetry"
```

Comment:
Is it possible to have this dynamically defined based on PR etc?

Reply:
With zuul, a new env is created each time.

However, if you want to use these playbooks locally (e.g. for recreating issues), it would be useful to have that separation.

Reply:
I'm more worried about implementing changes and only testing with `service-telemetry` and then finding that if someone uses an alternative namespace, that everything gets broken. I've definitely implemented something in the past that was caught by using different namespace values in the CI, and if possible I'd like to keep that test functionality.

It could be as simple as using `set_fact` and some `random()` function or whatever, and then using that as the namespace value across the test?

Reply:
Zuul definitely has some var that would be suitable. Maybe not PR number, but a build number or something similar

Reply:
FYI, you can find all the zuul vars in the logs tab in a build, under zuul-info/inventory.yaml
zuul.change is the PR number, and zuul.build is the build number (same as the URL below), I think they'll do nicely

e.g. https://review.rdoproject.org/zuul/build/8edc7f4a83a6463b9c40f9263af82e84/log/zuul-info/inventory.yaml